launch.go

```diff
@@ -270,19 +270,90 @@ func createMetaSpace(metaSpace string) error {
 }
 
 func writeMetafile(metaSpace, metaFile, metaLog string, mergedMeta map[string]interface{}) error {
-	metaByte := []byte("")
-	log.Println("Marshalling Merged Meta JSON")
-	metaByte, err := marshal(mergedMeta)
+	var metaPath = metaSpace + "/" + metaFile
+
+	_, err := os.Stat(metaPath)
+
+	// If file doesn't already exist, write to new file
+	if os.IsNotExist(err) {
+		metaByte := []byte("")
+		log.Println("Marshalling Merged Meta JSON in writeMetafile")
+		metaByte, err := marshal(mergedMeta)
+
+		if err != nil {
+			return fmt.Errorf("Parsing Meta JSON: %v", err)
+		}
+
+		err = writeFile(metaPath, metaByte, 0666)
+		if err != nil {
+			return fmt.Errorf("Writing Parent %v Meta JSON: %v", metaLog, err)
+		}
+	} else {
```

Comment:
will this ever happen ? It's a new build, and no external fetch has happened yet ?